apps/teams-test-app/src/components/PagesAPIs.tsx

```diff
@@ -33,6 +31,35 @@ const ReturnFocus = (): React.ReactElement =>
     },
   });
 
+const SetCurrentFrame = (): React.ReactElement =>
+  ApiWithTextInput<FrameInfo>({
+    name: 'setCurrentFrame',
+    title: 'Set current frame',
+    onClick: {
+      validateInput: input => {
+        if (!input.websiteUrl || !input.contentUrl) {
+          throw new Error('websiteUrl and contentUrl are required.');
+        }
+      },
+      submit: async input => {
+        pages.setCurrentFrame(input);
+        return 'called';
+      },
+    },
+  });
+
+const RegisterFullScreenChangeHandler = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'registerFullScreenChangeHandler',
+    title: 'Register Full Screen Change Handler',
+    onClick: async setResult => {
+      pages.registerFullScreenHandler((isFullScreen: boolean): void => {
+        setResult('successfully called with isFullScreen:' + isFullScreen);
+      });
+      return 'registered';
```

Comment:
Should this actually return a noHostSdkMsg? I think that's what I've seen with the other functions, a noHubSdkMsg returned for display if the callback isn't called from the API call.

Reply:
There aren't any tests that validate what is displayed here, so I figured we can go with 'registered' which seems more informative than '... was called, but there was no response from the Host SDK'.

Reply:
Discussed offline. The host is not supposed to response to the registration itself, so showing 'no response from the Host SDK' is kind of weird, because no response is expected at that time.